### ISSUE TITLE
Data protection / privacy 

### ISSUE THREAD

**Issue by desputinski** [tcfev/forDem#117]
It is important, that forDem respects personal data an privacy. So let's try to define, what that means.

- Respect European data protection regulation (DSGVO): https://en.wikipedia.org/wiki/General_Data_Protection_Regulation
- Try Don't use any services, that communicate with servers sitting in the USA, such as Google Fonts, Google Analytics, Facebook-Share-Buttons etc.
- We don't use Google- or Apple Maps etc.

Internal communication

- Don't use commercial social media platforms or apps or chat- or video call apps (e.g. Facebook, Whatsapp, Zoom, MS Teams etc.)

**Comment by desputinski** [comment]
Any other ideas or input or different opinions on that?

**Comment by armantorkzaban** [comment]
Item number 1 is good, however I am not sure how we can avoid U.S companies. 
This needs more detail.

I give one example of where it might go wrong: 
Users want to connect their social media accounts, import or export things from/to them etc. Related: #139 #112

**Comment by lukas-h** [comment]
You are right, that it is really hard to avoid US alltogether. But it is important not to have many third-party services opening a network connection to our applications.
It is important to host all of the fundamental parts of the software on the first-party servers. This means, all the html, css, js, fonts, assets (maybe), and especially the canvaskit scripts for flutter, as they are by default hosted on a 3rd-party cdn

**Comment by desputinski** [comment]
I agree. And it is important to avoid google fonts, there is a lot of problems with that in Germany right now, because there are specialized loyers, who sue private persons and institutions vor DSGO-Violations of https://en.wikipedia.org/wiki/General_Data_Protection_Regulation
